meerkat_frontend/src/js/technical/tableManager.js

```diff
@@ -1209,150 +1209,292 @@ function drawAllClinicsCompleteness(containerID, regionID, locations, data) {
-                width: "10%",
-                valign: "middle"
-            },
-            {
-                field: "district",
-                title: i18n.gettext('District'),
-                width: "10%",
-                valign: "middle"
-            },
-            {
-                field: "clinic",
-                title: i18n.gettext('Clinic'),
-                'searchable': true,
-                width: "10%",
-                valign: "middle"
-            },
-            {
-                field: "report_date",
-                title: i18n.gettext('Date <br /> Reported'),
-                width: "10%",
-                valign: "middle"
-            },
-            {
-                field: "investigation_date",
-                title: i18n.gettext('Date <br /> Investigated'),
-                valign: "middle"
-            },
-            {
-                field: "status",
-                title: i18n.gettext('Status'),
-                valign: "middle"
-
-            },
-            {
-                field: "age",
-                title: i18n.gettext('Age'),
-                valign: "middle"
-
-            },
-            {
-                field: "gender",
-                title: i18n.gettext('Gender'),
-                valign: "middle"
-            },
-            {
-                field: "profession",
-                title: i18n.gettext('Profession'),
-                valign: "middle"
-            },
-            {
-                field: "status_2",
-                title: i18n.gettext('Status'),
-                valign: "middle"
-
-            }
-        ];
-        var data = [];
-
-        for (var i in cases) {
-            c = cases[i];
-
-            var datum = {
-                alert_id: c.variables.alert_id,
-                region: i18n.gettext(locations[c.region].name),
-                district: i18n.gettext(locations[c.district].name),
-                report_date: c.date.split("T")[0],
-
-                age: c.variables.agv_1
-
-            };
-            if ("ale_1" in c.variables) {
-                datum.investigation_date = c.variables.ale_1.split("T")[0];
-            }
-            if (c.clinic) {
-                datum.clinic = i18n.gettext(locations[c.clinic].name);
-            } else {
-                datum.clinic = i18n.gettext(c.type_name);
-            }
-            if (c.variables.pla_2) {
-                datum.profession = i18n.gettext(c.variables.pla_2);
-            } else {
-                datum.profession = "";
-            }
-            var gender = "";
-
-            if ("gen_1" in c.variables) {
-                datum.gender = i18n.gettext("Female");
-            } else if ("gen_2" in c.variables) {
-                datum.gender = i18n.gettext("Male");
-            }
+    var columns = [{
+      field: "alert_id",
+      title: i18n.gettext('Alert ID'),
+      'searchable': true,
+      width: "10%",
+      valign: "middle"
+    },
+		   {
```

Comment:
Indentation is quite strange here.  Trying to move all this code over to 4 hard spaces like python.  Not that we are going to acheive it any time soon.